chainstate/tx-verifier/src/transaction_verifier/mod.rs

```diff
@@ -337,17 +359,13 @@ impl<'a, S: TransactionVerifierStorageRef> TransactionVerifier<'a, S> {
             None => return Ok(()),
         };
 
-        for (input_idx, input) in inputs.iter().enumerate() {
+        for (_input_idx, input) in inputs.iter().enumerate() {
```

Comment:
```suggestion
        for input in inputs.iter() {
```

Reply:
I think `inputs.iter()` could be just `&iter` but not sure that would be an improvement